src/order/bounded_lattice.lean

```diff
@@ -512,10 +512,11 @@ instance has_lt [has_lt α] : has_lt (with_bot α) :=
   @has_lt.lt (with_bot α) _ (some a) (some b) ↔ a < b :=
 by simp [(<)]
 
-lemma bot_lt_some [has_lt α] (a : α) : (⊥ : with_bot α) < some a :=
+lemma none_lt_some [has_lt α] (a : α) :
+  @has_lt.lt (with_bot α) _ none (some a) :=
```

Comment:
what's the point of keeping this one? I mean, the idea is to use the standard form (so, bot and coe), as in the next one. This one is breaking the abstraction barrier, and should probably be removed.

Same thing for the other one below.

Reply:
Sure. Shall I also remove `some_lt_none`, then?

Reply:
Note however that `bot_lt_coe` doesn't apply to, say, `\bot < 0` in `with_bot \N` because it doesn't recognize `0` as `↑0`, which is annoying. For that reason, I'm rather for keeping them.

Reply:
ok, let's keep them, it doesn't hurt so much.